routingpy/isochrone.py

```diff
@@ -64,9 +64,13 @@ def __init__(self, geometry=None, interval=None, center=None):
     @property
     def geometry(self):
         """
-        The geometry of the isochrone as [[lon1, lat1], [lon2, lat2], ...] list.
+        The geometry of the isochrone as [[[lon1, lat1], [lon2, lat2], ...]] list.
 
         :rtype: list or None
+
+        .. note::
+           Since it's not known whether providers' responses adhere to OGC standards, caution is advised with regard
+           to possible orientation issues of Polygons.
```

Comment:
```suggestion
        .. note::
           Since it's not known whether providers' responses adhere to OGC standards, caution is advised with regard
           to possible orientation issues of exterior and interior rings of the resulting polygons.
```